frontend/src/utilities.js

```diff
@@ -32,6 +32,8 @@ import {
 import { ClassificationDropdown } from './components';
 
 
+
+
```

Comment:
too many spaces :smiley:

Reply:
sometimes I want to ask eslint what it does around here